Danki2/Assets/Scripts/Abilities/InstantCast/Fireball.cs

```diff
@@ -2,27 +2,27 @@
 
 public class Fireball : InstantCast
 {
-    private static readonly float FIREBALL_SPEED = 5;
-    private readonly AbilityContext _context;
+    private static readonly float SPEED = 5;
+    private static readonly Vector3 POSITION_TRANSFORM = new Vector3(0, 1.25f, 0);
```

Comment:
By our naming convention, these private fields should be _fireballSpeed and _positionTransform.

Reply:
Sure thing